lib/marqeta/transaction.rb

```diff
@@ -3,6 +3,12 @@ class Transaction < ApiObject
     PENDING_STATE = 'PENDING'.freeze
     DECLINED_STATE = 'DECLINED'.freeze
 
+    DECLINED_BY_JIT_RESPONSE_CODE = '402'.freeze
```

Comment:
Does the API actually return strings rather than integers?

Reply:
@andyw8 yea, they even have codes like '0002'
https://www.marqeta.com/api/docs/Vh7kRx0AAMcA17eh/transactions#transaction_response_codes